### PR TITLE
Add Eventy hooks for before/after rapidez indexer

### PR DESCRIPTION
**Pull request by Jade-GG**
Simple solution to being able to quickly hook in multiple actions around the indexer

Can be used like this (e.g. in `App\Providers\AppServiceProvider:boot`):

```php
Eventy::addAction('index.before', function($context) {
    $context->call('rapidez:index:sortings');
});
Eventy::addAction('index.after', function($context) {
    $context->call('rapidez:index:additionals');
});
```

### CODE REVIEW

File: src/Commands/IndexProductsCommand.php [hunk 1]
```diff
@@ -21,6 +21,8 @@ class IndexProductsCommand extends InteractsWithElasticsearchCommand
 
     public function handle()
     {
+        Eventy::action('rapidez.index.before', $this);
```
Comment (royduin) [inline src/Commands/IndexProductsCommand.php:24]:
```suggestion
        Eventy::action('index.before', $this);
```

File: src/Commands/IndexProductsCommand.php [hunk 2]
```diff
@@ -86,6 +88,8 @@ public function handle()
             $bar->finish();
             $this->line('');
         }
+        
+        Eventy::action('rapidez.index.after', $this);
```
Comment (royduin) [inline src/Commands/IndexProductsCommand.php:92]:
```suggestion
        Eventy::action('index.after', $this);
```

**Review by royduin — CHANGES_REQUESTED** [review]
Please add it to the docs and I also like to have this in 0.x

**Comment by Jade-GG** [comment]
> Please add it to the docs and I also like to have this in 0.x

I have made a PR: https://github.com/rapidez/docs/pull/31

**Comment by royduin** [comment]
Merged, could you also make a PR to the 0.x branch?

**Pull request merged**